lib/utils.php

```diff
@@ -10,12 +10,18 @@
  * It is equivalent to want lodash get provides for JavaScript and is useful to have something similar
  * in php so functions that do the same thing on the client and sever can have identical code.
  *
- * @param array $array  An array from where we want to retrieve some information from.
- * @param array $path   An array containing the path we want to retrieve.
+ * @param array $array    An array from where we want to retrieve some information from.
+ * @param array $path     An array containing the path we want to retrieve.
+ * @param array $default  The return value if $array or $path is not expected input type.
  *
  * @return array Containing a set of css rules.
  */
-function gutenberg_experimental_get( $array, $path ) {
+function gutenberg_experimental_get( $array, $path, $default = array() ) {
+	// Confirm input values are expected type to avoid notice warnings.
+	if ( ! is_array( $array ) || ! is_array( $path ) ) {
+		return $default;
+	}
+
 	$path_length = count( $path );
 	for ( $i = 0; $i < $path_length; ++$i ) {
 		if ( empty( $array[ $path[ $i ] ] ) ) {
```

Comment:
Can we also return the default within this loop? There's one function call that will need updating within `gutenberg_experimental_global_styles_flatten_styles_tree`; I think in that case we can make `null` the default.

Reply:
👍 Missed that part, updated and fixed doc string too.